fix: Ensure cache does not store a value of 0

rubencarvalho opened the pull request:
### Description

<!-- Include a summary of the changes and the related issue. -->

<!-- Also include relevant motivation and context. -->

Related links, issue #, if available: n/a

### How has this been tested?

<!-- How did you test to verify your changes? -->

<!-- How can reviewers test these changes efficiently? -->

<details>
   <summary>Review checklist</summary>

_The following items are to be evaluated by the author(s) and the reviewer(s)._

#### Correctness

- _Changes include appropriate documentation updates._
- _Changes are backward-compatible if not indicated, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#public-apis)._
- _Changes do not include unsupported browser features, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#browsers-support)._
- _Changes were manually tested for accessibility, see [accessibility guidelines](https://cloudscape.design/foundation/core-principles/accessibility/)._

#### Security

- _If the code handles URLs: all URLs are validated through [the `checkSafeUrl` function](https://github.com/cloudscape-design/components/blob/main/src/internal/utils/check-safe-url.ts)._

#### Testing

- _Changes are covered with new/existing unit tests?_
- _Changes are covered with new/existing integration tests?_
</details>

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

jperals reviewed (approved):
Let's ship this fix but I think it should be followed by actual test coverage. The new dev page could be used for an integration test.

Also, minor: I'd prefer a PR description that better explains what this fixes for customers, something like: "fix: Prevent chart ticks from overlapping after being rendered in a collapsed element"

rubencarvalho commented:
> Let's ship this fix but I think it should be followed by actual test coverage. The new dev page could be used for an integration test.
> 
> Also, minor: I'd prefer a PR description that better explains what this fixes for customers, something like: "fix: Prevent chart ticks from overlapping after being rendered in a collapsed element"
I updated the description - I was so deep into the issue I forgot to give it a holistic description 😛 

The test page I added is to screenshot test on our internal integration tests :)
